app/src/main/java/de/grobox/transportr/trips/BaseViewHolder.kt

```diff
@@ -33,20 +33,20 @@ import de.schildbach.pte.dto.Position
 import de.schildbach.pte.dto.Stop
 import java.util.*
 
-internal abstract class BaseViewHolder(v: View, protected val listener: LegClickListener) : RecyclerView.ViewHolder(v) {
+internal abstract class BaseViewHolder(v: View) : RecyclerView.ViewHolder(v) {
 
     protected val context: Context = v.context
     protected val fromTime: TextView = v.findViewById(R.id.fromTime)
     protected val toTime: TextView = v.findViewById(R.id.toTime)
     protected val fromDelay: TextView = v.findViewById(R.id.fromDelay)
     protected val toDelay: TextView = v.findViewById(R.id.toDelay)
 
-    fun setArrivalTimes(timeView: TextView, delayView: TextView, stop: Stop) {
+    fun setArrivalTimes(timeView: TextView?, delayView: TextView, stop: Stop, showZeroDelay: Boolean = true) {
```

Comment:
Why do we need `showZeroDelay`? I think the convention was (or should be) to show a green +0 if real time information is available and to show nothing if it isn't available.

Reply:
Up to now, the green `+0` is only shown in the trip details view, not in the search results view (if real time information is available, obviously). So I have added this flag to be able to differentiate between the two cases. But if you want to change the app's behavior here, it would be easy to adopt.

Reply:
I think we should indicate the availability of real-time information everywhere consistently. So we could drop the flag, right?

Reply:
So then yes, the flag can be dropped.